test/resolvers/pil.js

```diff
@@ -139,8 +139,9 @@ describe('PIL resolver', () => {
           .then(pil => {
             assert.deepEqual(pil, undefined);
           })
-          .then(() => this.models.PIL.query())
+          .then(() => this.models.PIL.queryWithDeleted().findById(opts.id))
```

Comment:
Is this just a fix for something else that's wildly out of scope?

Reply:
Yeah, this was returning an empty array before, which then passed the assertion even though there was no PIL.